src/views/Farms/components/BCakeBoosterCard.tsx

```diff
@@ -0,0 +1,145 @@
+import { Button, Card, CardBody, CardFooter, Text, Box, Flex } from '@pancakeswap/uikit'
+import { useWeb3React } from '@web3-react/core'
+import ConnectWalletButton from 'components/ConnectWalletButton'
+import { useTranslation } from 'contexts/Localization'
+import { useBCakeFarmBoosterProxyFactoryContract } from 'hooks/useContract'
+import styled from 'styled-components'
+import { useBCakeProxyContractAddress } from '../hooks/useBCakeProxyContractAddress'
+import { useUserBoosterStatus } from '../hooks/useUserBoosterStatus'
+import { useUserLockedCakeStatus } from '../hooks/useUserLockedCakeStatus'
+
+export const CardWrapper = styled.div`
+  position: absolute;
+  top: 30px;
+  right: 25px;
+  width: 328px;
+`
+const StyledCardBody = styled(CardBody)`
+  border-bottom: none;
+`
+const StyledCardFooter = styled(CardFooter)`
+  border-top: none;
+  position: relative;
+  &::before {
+    content: '';
+    position: absolute;
+    height: 1px;
+    width: calc(100% - 48px);
+    top: 0px;
+    left: 24px;
+    background-color: ${({ theme }) => theme.colors.cardBorder};
+  }
+`
+
+export const BCakeBoosterCard = () => {
+  const { t } = useTranslation()
+  return (
+    <CardWrapper>
+      <Card>
+        <StyledCardBody>
+          <Text fontSize={22} bold color="text" marginBottom="-12px">
+            {t('Yield Booster')}
+          </Text>
+        </StyledCardBody>
+        <StyledCardFooter>
+          <CardContent />
+        </StyledCardFooter>
+      </Card>
+    </CardWrapper>
+  )
+}
+
+const CardContent: React.FC = () => {
+  const { t } = useTranslation()
+  const { account } = useWeb3React()
+  const farmBoosterProxyFactoryContract = useBCakeFarmBoosterProxyFactoryContract()
+  const { proxyCreated } = useBCakeProxyContractAddress(account)
+  const { maxBoostCounts, remainingCounts } = useUserBoosterStatus(account)
+  const { locked, lockedEnd } = useUserLockedCakeStatus()
+  useUserLockedCakeStatus()
+
+  if (!account)
+    return (
+      <Box>
+        <Text color="textSubtle" fontSize={12} bold mt="-12px">
+          {t('Connect wallet to view booster')}
+        </Text>
+        <Text color="textSubtle" fontSize={12} mb="16px">
+          {t('An active fixed-term CAKE staking position is required for activating farm yield boosters.')}
+        </Text>
+        <ConnectWalletButton />
```

Comment:
How about user hasn't licked cake yet? It should have a Go to pool button, right?